example/src/modules/forms/TextAreaScreen.tsx

```diff
@@ -0,0 +1,92 @@
+import React from "react";
```

Comment:
Can we have the TextArea Sizes x TextArea Theme in order?  That way it is easier to check. @crstnmac

Reply:
Sorted the components

Reply:
Great, Can you use a Radio component to switch between sizes and variants, and just one TextArea on the screen? That way it's neater.

Reply:
<image src="https://user-images.githubusercontent.com/20360870/201634352-968b1ae4-356d-47ef-8475-4de09a9fb9dd.png" height="800" width="350" />